TS_TSX.md

```diff
@@ -0,0 +1,48 @@
+# File Metrics for TypeScript (TS) and TypeScript JSX (TSX)
+
+This document outlines the definition of file metrics for TypeScript (TS) and TypeScript JSX (TSX).
+
+### 1. classes
```

Comment:
We do not have numbers for them in the main Readme. So I think, we should not do this here, either. Also, the order of the metrics here does not match that of the readme. We should probably order them identical to the readme here.